src/App.css

```
@@ -35,41 +83,247 @@ input[type=text], input[type=password], input[type=email] {
     padding: 12px 15px;
 }
 
-.submit-btn-login {
+button[type=submit] {
+    color: var(--red);
     background-color: var(--primary);
-    color: var(--dark);
+    color: var(--red);
     padding: 10px 10px;
     cursor: pointer;
-    margin: 8px 0;
+    margin: 10px 0;
     border: none;
     width: 100%;
+    font-weight: 600;
 }
 
 .submit-btn-login:hover{
     opacity: 0.8;
+    transition: 0.3s;
 }
 
 .login-inner-text {
     font-size: 20px;
     text-align: center;
 }
 
+.login-message {
+    background-color: plum;
+}
+
+.login-error {
+    background-color: rgb(189, 51, 51);
+    text-align: center;
+}
+
 /* for rgistration */
-.reg-btn {
+.reg-btn:hover {
+    opacity: 0.8;
+    transition: 0.3s;
+}
+.register-inner-text {
+    font-size: 20px;
+    text-align: center;
+}
+.reg-error {
+    background-color: rgb(189, 51, 51);
+    text-align: center;
+}
+
+/* for Home  */
+.home-header {
+    text-align: center;
+    box-shadow: 1px 1.5px 9px 0.2px;
+    border-radius: 15px;
+    margin: 10rem 17px;
+    padding: 15px;
+}
+
+.home-header h2 {
+    color: var(--blue);
+    font-size: 2rem;
+}
+
+.home-header h1 {
+    color: var(--blue-alt);
+    font-size: 3rem;
+    margin-bottom: 1rem;
+}
+
+.home-header h1 span {
+    color: var(--primary-alt);
+}
+
+.home-header p {
+    font-weight: 500;
+    color: var(--dark-alt);
+}
+
+/* search bar */
+.search-bar-wrap > form {
     background-color: var(--primary);
-    color: var(--dark);
-    padding: 10px 10px;
-    cursor: pointer;
-    margin: 8px 0;
     border: none;
+    width: fit-content;
+    margin: 2.5rem auto 4rem auto;
```

You can use 
```suggestion
margin: 2.5rem auto 4rem
```
also

---

ok